client/web/compose/src/components/ModuleFields/Editor/Bool.vue

```diff
@@ -1,26 +1,40 @@
 <template>
   <b-form-group
-    label-class="d-flex align-items-center text-primary p-0"
+    :label-cols-md="horizontal && '5'"
+    :label-cols-xl="horizontal && '4'"
+    :content-cols-md="horizontal && '7'"
+    :content-cols-xl="horizontal && '8'"
   >
     <template
       v-if="field.options.switch"
       #label
     >
-      {{ label }}
+      <div
+        v-if="!valueOnly"
+        class="d-flex align-items-center text-primary p-0"
+      >
+        <span
+          class="d-inline-block text-truncate mw-100 pt-0"
+          :class="{ 'pb-1': !horizontal }"
```

Comment:
I might be miss understanding this (I can't run the code at the moment), but I'm not sure I'm a big fan of using padding to achieve a structural change (aligning horizontal vs vertical).